synapse/lib/stormsvc.py

```diff
@@ -182,6 +182,19 @@ async def _onTeleLink(self, proxy):
         names = [c.rsplit('.', 1)[-1] for c in clss]
 
         if 'StormSvc' in names:
+            rver = proxy._getSynVers()
+            if rver < (2, 8, 0):
+                mesg = f'Storm service {self.name} ({self.iden}) is running an older ' \
+                       f'version of synapse {rver}, please update to ensure proper function!'
+                logger.exception(mesg)
+
+                self.proxy._t_conf['retrysleep'] = 2.0
+
+                try:
+                    await proxy.newSynapseVersionRequiredPleaseUpdateServiceTo290()
```

Comment:
what is this? this makes no sense

Reply:
The idea here was to make a storm service running an older version blow up and print out some kind of an error message:
```synapse.exc.NoSuchMeth: NoSuchMeth: name='newSynapseVersionRequiredPleaseUpdateServiceTo290'```
I'm open to better ideas.

Reply:
Why are we doing this? This is forcing flag day for something that you are not actually enforcing as a schema requirement for a storm package.

Reply:
Trying to avoid a service being silently broken, I guess we could make specifying a minversion a requirement and check against that instead of the synapse version.

Reply:
Removed this stuff because it doesn't seem necessary anymore